assets/css/components/forms.scss

```
@@ -0,0 +1,55 @@
+.input {
+  @extend %font-ui-text-base-oxygen;
+  color: $color-white-150;
+  background-color: $color-slate-800;
+  border-style: none;
+  border-radius: 3px;
+  transition: all 0.3s ease-in-out;
+
+  &:hover {
+    background-color: $color-slate-900;
+  }
+
+  &:focus {
+    box-sizing: border-box;
+    outline: none;
+    box-shadow: 0 0 0 2px $color-focus;
+  }
+
+  &:active {
+    color: $color-white-180;
+  }
+
+  &:disabled {
+    cursor: not-allowed;
+    opacity: 0.4;
+  }
+
+  .input--error {
```

```suggestion
  &--error {
```
You can use the `&` here instead of typing out `.input` again.